cli-feed-v3.json

```diff
@@ -282,6 +282,17 @@
       "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/1.0.3.10182.zip",
       "sha2": "3480E4AC8B8C739AC6BA176468B4CCA4EB2D333D8F1D21CB2A10FEBE2C3C3BF1",
       "FUNCTIONS_EXTENSION_VERSION": "~1"
+    },
+    "1.4.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.15",

```

Comment:
nit: Latest Net SDK is 1.0.19, may not make a difference since it doesn't have any updates for v1

Reply:
is it safe to update this to 1.0.19?

Reply:
nevermind, I realized I uped it to a random version. Updated to 1.0.19. Thanks for catching that.